__init__.py

```diff
@@ -121,27 +132,34 @@ def contentdir(self):
 
 
 class SpicyAnalyzer(zeekpkg.template.Feature):
-    """Feature for a Spicy-based analyzer."""
-    def name(self):
-        return 'spicy-analyzer'
+    """Base class for Spicy-based analyzers."""
 
     def contentdir(self):
         return os.path.join('features', self.name())
 
     def needed_user_vars(self):
-        """Specify required user variables."""
-        return ['name', 'namespace']
+        return ['name', 'analyzer']
 
     def validate(self, tmpl):
         """Validate feature prerequisites."""
-        for parameter in ['name', 'ns']:
+        for parameter in self.needed_user_vars():
             value = tmpl.lookup_param(parameter)
             if not value or len(value) == 0:
                 raise zeekpkg.template.InputError(
                         'package requires a {}'.format(parameter))
 
     def instantiate(self, tmpl):
-        super().instantiate(tmpl)
+        # Instead of calling super(), do this ourselves to instantiate symlinks as files.
+        for orig_file, path_name, file_name, content in self._walk(tmpl):
+            if os.path.islink(orig_file):
+                with open(orig_file, 'rb') as hdl:
+                    content = self._replace(tmpl, hdl.read())
+
+            self.instantiate_file(tmpl, orig_file, path_name, file_name, content)
+
+        # Remove any files marked as unneeded.
+        for p in glob.glob(os.path.join(self._packagedir, "**/*.REMOVE"), recursive=True):
+               os.unlink(p)
```

Comment:
Minor nit: pylint flagged that indentation is off here.

Reply:
Fixed.